File: Exercises/rb101-rb109_small_problems/easy_8/double_char_1.rb
def repeater(string)
  new_str = ''
  string.each_char { |char| new_str << char * 2 }
  new_str
end

puts repeater('Hello') == "HHeelllloo"
puts repeater("Good job!") == "GGoooodd  jjoobb!!"
puts repeater('') == ''

File: js210/exercises/easy_5/double_char_1.js
function repeater(string) {
  let doubleString = '';

  for (let index = 0; index < string.length; index += 1) {
    let currentChar = string[index];
    doubleString = doubleString.concat(currentChar, currentChar);
  }

  return doubleString;
}

console.log(repeater('Hello'));        // "HHeelllloo"
console.log(repeater('Good job!'));    // "GGoooodd  jjoobb!!"
console.log(repeater(''));             // ""
